scripts/tests/test_zkai_native_attention_mlp_rmsnorm_input_fused_adapter_gate.py

```
@@ -0,0 +1,58 @@
+        self.assertEqual(payload["summary"]["fused_adapter_trace_cells"], 0)
+        cases = payload["mutation_result"]["cases"]
+        self.assertEqual([case["name"] for case in cases], list(gate.MUTATION_NAMES))
+        self.assertTrue(all(case["rejected"] for case in cases))
+
+    def test_frontier_overclaim_is_rejected(self) -> None:
+        context = gate.build_context()
+        payload = gate.build_payload(context)
+        candidate = copy.deepcopy(payload)
+        candidate["comparisons"]["fused_vs_two_proof_frontier"]["frontier_win_claimed"] = True
+        candidate["payload_commitment"] = gate.payload_commitment(candidate)
+        with self.assertRaises(gate.RmsnormInputFusedAdapterGateError):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)